src/modules/vtol_att_control/standard.cpp

```diff
@@ -272,16 +260,16 @@ void Standard::update_transition_state()
 
 		// do blending of mc and fw controls if a blending airspeed has been provided and the minimum transition time has passed
 		if (_airspeed_trans_blend_margin > 0.0f &&
-		    PX4_ISFINITE(_airspeed_validated->calibrated_airspeed_m_s) &&
-		    _airspeed_validated->calibrated_airspeed_m_s > 0.0f &&
-		    _airspeed_validated->calibrated_airspeed_m_s >= _params->airspeed_blend &&
+		    PX4_ISFINITE(get_filtered_airspeed()) &&
+		    get_filtered_airspeed() > 0.0f &&
+		    get_filtered_airspeed() >= _params->airspeed_blend &&
 		    time_since_trans_start > getMinimumFrontTransitionTime()) {
```

Comment:
Don't know what I like better out of calling getter multiple times and making a temp variable.

Reply:
I'd say temp variable is more readable and robust, it's easier to see that they're the same variable. Maybe in theory the get_filtered_airspeed() could change between calls?

Reply:
The module is in a single thread, but not super-easy to see without a temp variable. Agree